spec/till_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'till'
+require 'timecop'
+
+describe Till do
+
+  let(:till) { described_class.new( { shopName: "The Coffee Connection",
+                                      address: "123 Lakeside Way",
+                                      phone: "16503600708",
+                                      prices: { "Cafe Latte"=> 4.75, "Flat White"=> 4.75 }
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)